redbot/core/dev_commands.py

```diff
@@ -28,7 +28,7 @@
 
 _ = Translator("Dev", __file__)
 
-START_CODE_BLOCK_RE = re.compile(r"^((```py)(?=\s)|(```))")
+START_CODE_BLOCK_RE = re.compile(r"^((```py)(?=\s)|(```python)(?=\s)|(```))")
```

Comment:
This can be simplified:
```suggestion
START_CODE_BLOCK_RE = re.compile(r"^((```py(thon)?)(?=\s)|(```))")
```